src/alr/alr-commands.adb

```diff
@@ -44,6 +45,7 @@ with Alr.Commands.Search;
 with Alr.Commands.Settings;
 with Alr.Commands.Show;
 with Alr.Commands.Test;
+with Alr.Commands.Test.Runner;
```

Comment:
`Alr.Commands.Test` is redundant with `Alr.Commands.Test.Runner`.

Reply:
Is it? `alr test` and `alr test-runner` are two different commands

Reply:
> Is it? alr test and alr test-runner are two different commands.

Yes, withing a child gives you visibility of all its parents. It's a minor nitpick, but still you never know when a future GNAT with all warnings enabled is going to bite you. (Actually I thought this was already caught by some GNAT flag.)

Reply:
I didn't know that, thanks!

The `Test.Runner` command will be gone in the commits I'm currently preparing, so this will be solved anyway :)